xwiki-platform-core/xwiki-platform-scheduler/xwiki-platform-scheduler-ui/src/main/resources/XWiki/SchedulerJobSheet.xml

```diff
@@ -20,67 +20,69 @@
  * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
 -->
 
-<xwikidoc version="1.1">
+<xwikidoc version="1.3" reference="XWiki.SchedulerJobSheet" locale="">
   <web>XWiki</web>
   <name>SchedulerJobSheet</name>
   <language/>
   <defaultLanguage/>
   <translation>0</translation>
   <creator>xwiki:XWiki.Admin</creator>
-  <creationDate>1252454400000</creationDate>
+  <creationDate>1252450800000</creationDate>
   <parent>XWiki.SchedulerJobClass</parent>
   <author>xwiki:XWiki.Admin</author>
   <contentAuthor>xwiki:XWiki.Admin</contentAuthor>
-  <date>1418224809000</date>
-  <contentUpdateDate>1418224809000</contentUpdateDate>
+  <date>1529424829000</date>
+  <contentUpdateDate>1529424829000</contentUpdateDate>
   <version>1.1</version>
   <title>#sheetTitle('Scheduler Job Sheet', 'XWiki.SchedulerJobClass')</title>
   <comment/>
   <minorEdit>false</minorEdit>
   <syntaxId>xwiki/2.1</syntaxId>
   <hidden>true</hidden>
   <content>{{velocity}}
-#if($doc.getObject('XWiki.SchedulerJobClass'))
-#set($job = $doc.getObject("XWiki.SchedulerJobClass"))
-**$services.localization.render('xe.scheduler.job.name')**
+#if ($doc.getObject('XWiki.SchedulerJobClass'))
+  #set ($job = $doc.getObject("XWiki.SchedulerJobClass"))
+  #macro (displayJobProperty $obj $propName)
+    #set ($class = $obj.xWikiClass)
+    #set ($prettyName = $class.get($propName).translatedPrettyName)
+    ; #if ($xcontext.action == 'edit')
```

Comment:
Not sure why you are putting `xform` only on edit mode. For example, if you look at an User Profile, the labels styling is present also in View mode. 
So I would remove the `#if / #else`.